android/engine/src/main/java/org/smartregister/fhircore/engine/ui/questionnaire/CustomPhotoCaptureFactory.kt

```diff
@@ -75,7 +78,11 @@ class CustomPhotoCaptureFactory(
       if (result) {
         lifecycleScope
           .launch(dispatcher.io()) {
-            imageFile = compress(context, imageFile)
+            imageFile =
+              compress(context, imageFile) {
```

Comment:
why do we compress it all here, how does that help with viewing it?

Reply:
because we need to reduce the image size before uploading it as a Base64 binary data.
so that when viewing the image as a QuestionnaireResponse, the image size is small enough that Android won't throw TransactionTooLargeException.